src/App.tsx

```diff
@@ -154,22 +161,41 @@ export const App = () => {
     ]);
   }, [handleCancel, messageHistoryStore, modelConfig.systemPrompt]);
 
+  const isMobile = useIsMobile();
+  const [burgerToggle, setBurgerToggle] = useState(false);
```

Comment:
I think if this is named something like `[chatHistoryOpen, setChatHistory]`, it makes its function a bit more clear.